From 7b0967aafd59b9f339c1c77dd0f65fb6325b9728 Mon Sep 17 00:00:00 2001
From: Dylan Baker <dylan@pnwbakers.com>
Date: Tue, 14 Aug 2018 10:24:29 -0700
Subject: [PATCH 16/27] scons: Require python 2.7

less than 2.7 is not supported.

v2: - Remove check for python >= 2.0, since we've already enforced 2.7

Reviewed-by: Eric Engestrom <eric.engestrom@intel.com>
Reviewed-by: Jose Fonseca <jfonseca@vmware.com>
---
 SConstruct       | 1 +
 scons/gallium.py | 4 ----
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/SConstruct b/SConstruct
index 6e034fb968..51dc301a9a 100644
--- a/SConstruct
+++ b/SConstruct
@@ -31,6 +31,7 @@ import common
 # Minimal scons version
 
 EnsureSConsVersion(2, 4)
+EnsurePythonVersion(2, 7)
 
 
 #######################################################################
diff --git a/scons/gallium.py b/scons/gallium.py
index 92a762a0c1..659da72c1c 100755
--- a/scons/gallium.py
+++ b/scons/gallium.py
@@ -221,10 +221,6 @@ def generate(env):
     env['suncc'] = env['platform'] == 'sunos' and os.path.basename(env['CC']) == 'cc'
     env['icc'] = 'icc' == os.path.basename(env['CC'])
 
-    if env['msvc'] and env['toolchain'] == 'default' and env['machine'] == 'x86_64':
-        # MSVC x64 support is broken in earlier versions of scons
-        env.EnsurePythonVersion(2, 0)
-
     # shortcuts
     machine = env['machine']
     platform = env['platform']
-- 
2.18.1

